library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -165,6 +165,9 @@ $skin-with-icons: true !default;
 $icon-size: 20px !default;
 $tool-menu-icon-on-left: true !default;
 
+/* Icon Overrides */
```

Comment:
Also most will look to the defaults file for what can be configured which is why I added it here.